mmv1/third_party/terraform/tpgresource/utils_test.go

```diff
@@ -37,6 +37,145 @@ var fictionalSchema = map[string]*schema.Schema{
 	},
 }
 
+func TestSortByConfigOrder(t *testing.T) {
+	cases := map[string]struct {
+		configData, apiData []string
+		want                []string
+		wantError           bool
+	}{
+		"empty config data and api data": {
+			configData: []string{},
+			apiData:    []string{},
+			want:       []string{},
+		},
+		"config data with empty api data": {
+			configData: []string{"one", "two"},
+			apiData:    []string{},
+			want:       []string{},
+		},
+		"empty config data with api data": {
```

Comment:
Nit: why does the order change here?
I know this does not affect the results.

Reply:
Currently, we sort the new items alphabetically. It's not strictly necessary but may make things easier to read. (It also matches the current instance group manager logic.)

Reply:
OK, Understood. LGTM